### PR TITLE
Use Python 3, instead of the default of the machine

### PR DESCRIPTION
**Pull request by eduardoj**
Let's try with Python 3, and only fall back explicitly to Python 2 if some playbook cannot be run.

This also prevents this warning from appearing on each `ansible` run:

```
[WARNING]: Platform linux on host obs is using the discovered Python interpreter at /usr/bin/python,
but future installation of another Python interpreter could change this. See
https://docs.ansible.com/ansible/2.9/reference_appendices/interpreter_discovery.html for more information.
```

### CODE REVIEW

File: inventory/production.yml [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
 all:
-  vars:
-    ansible_python_interpreter: auto
   hosts:
     obs:
       ansible_connection: ssh
       # configured host in your ssh config
       ansible_host: obs
-      apache_sysconfig_regex_unset: 'APACHE_SERVER_FLAGS="STATUS"' 
+      apache_sysconfig_regex_unset: 'APACHE_SERVER_FLAGS="STATUS"'
```
Comment (vpereira) [inline inventory/production.yml:8]: is that change related with the change with python2/python3?

**Review by vpereira — CHANGES_REQUESTED** [review]

File: inventory/production.yml (hunk 1 above)
Comment (eduardoj) [reply inventory/production.yml:8]: Nop. Should I separate it in another commit?

Comment (eduardoj) [reply inventory/production.yml:8]: Ops, you edited your comment, and added another one. 

> Did you test manually if it's working?

Yes, sure.

Comment (vpereira) [reply inventory/production.yml:8]: > Nop. Should I separate it in another commit?

No not necessary, I just wanted to know if it was something commited by accident

Comment (eduardoj) [reply inventory/production.yml:8]: > is that change related with the change with python2/python3?

I created another commit.

Comment (eduardoj) [reply inventory/production.yml:8]: > > Nop. Should I separate it in another commit?
> 
> No not necessary, I just wanted to know if it was something commited by accident

Ups, late.

**Review by vpereira — APPROVED** [review]
